drivers/CoreVideo/tests/CoreFont_test.cpp

```diff
@@ -143,7 +144,7 @@ TEST_F(CoreFontTest, drawCharacterWithColor)
 {
 	// ENHANCEMENT: Set pixels_lit by checking number of bit set.
 
-	CoreFont::Character character;
+	Character character {};
 	character.ascii			 = '.';
 	CGColor foreground_color = CGColor::pure_red;
 	CGColor background_color = CGColor::black;
```

Comment:
building on the previous comment, storing the colors inside the struct would allow us to do:

```suggestion
	Character character {.ascii = '.',
						 .foreground = CGColor::pure_red,
						 .background= CGColor::black,
						};
```